buildbot/buildbot/server.py

```diff
@@ -33,6 +33,8 @@
     state.HLS_FINISH: "Synthesized",
     state.MAKE: "make",
     state.MAKE_PROGRESS: "make-ing",
+    state.AFI_START:"starting AFI generation",
+    state.AFI:"generating AFI",
```

Comment:
Following the pattern of the other names here, maybe these could be called "Starting AFI" and "Generating AFI." (Also, this is a tiny thing, but it's customary to put a space after the `:`. :smiley:)